### ISSUE TITLE
MySql Command is override on start

### ISSUE THREAD

**Issue by AlissonMedeiros** [testcontainers/testcontainers-java#534]
When I start a new MySql container I set a diferent command, but this is override per the method "configure()"

My code:
```java
(MySQLContainer) new MySQLContainer()
                .withDatabaseName("f"+timeStamp)
                .withUsername("f"+timeStamp)
                .withPassword("f"+timeStamp)
                .withCommand("mysqld", "--lower_case_table_names=1")
                .withLogConsumer(new Slf4jLogConsumer(logger));
```

The code of MYSQLContainer:

```java
    @Override
    protected void configure() {
        optionallyMapResourceParameterAsVolume(MY_CNF_CONFIG_OVERRIDE_PARAM_NAME, "/etc/mysql/conf.d", "mysql-default-conf");

        addExposedPort(3306);
        addEnv("MYSQL_DATABASE", databaseName);
        addEnv("MYSQL_USER", username);
        addEnv("MYSQL_PASSWORD", password);
        addEnv("MYSQL_ROOT_PASSWORD", "test");
        setCommand("mysqld");
        setStartupAttempts(3);
    }
```
I need this to support case insentive. Do you know athoner way to do this? If not I can fix and open PR.

Another option is add **lower_case_table_names=1"** in the file **my.cnf** Or have
a way to add more configurations in the file.
 
Many thanks.

**Comment by AlissonMedeiros** [comment]
****

**Comment by bsideup** [comment]
Hi @AlissonMedeiros,

That's a bug indeed. For now, you can override `MySQLContainer`'s `configure` command and set your command there. PR is also highly appreciated (you can just move `setCommand` to constructor) :)

**Comment by AlissonMedeiros** [comment]
@bsideup Thanks for help, I try to do this.

**Comment by pauldailly** [comment]
Hi there, would I be correct in saying the MariaDb test container module also suffers from this same issue (see https://github.com/testcontainers/testcontainers-java/blob/8402356b2784734010c445949eff76ae69159a1e/modules/mariadb/src/main/java/org/testcontainers/containers/MariaDBContainer.java#L42)?

If so, I'd be happy to start working on a PR similar to #560 if you would like me to?

One thing I am unsure of is the fact there's both a [mariadb module](https://github.com/testcontainers/testcontainers-java/tree/master/modules/mariadb) in this project and a separate [testcontainers-java-module-mariadb](https://github.com/testcontainers/testcontainers-java-module-mariadb) repository too. To which repo would I submit the PR?

**Comment by kiview** [comment]
@pauldailly 
Yes, you are right. The correct module is the one in the main repository. We've moved all modules to the main repo at one point in time (mono repo). PR very much appreciated :slightly_smiling_face:

**Comment by rnorth** [comment]
Released in [1.8.1](https://github.com/testcontainers/testcontainers-java/releases/tag/1.8.1)! 🎉

Thanks for flagging this, @AlissonMedeiros and @pauldailly.
